packages/shared/src/components/squads/SquadMemberAuthorBadge.tsx

```diff
@@ -41,12 +44,22 @@ const SquadMemberRoleBadge = ({
 
   return (
     <UserBadge
-      className={classNames('text-cabbage-40', className)}
+      className={classNames('text-theme-color-cabbage', className)}
       content={role}
       Icon={Icon}
       iconProps={iconProps}
     />
   );
 };
 
-export default SquadMemberRoleBadge;
+const SquadMemberAuthorBadge = ({
```

Comment:
Think we should abstract to 2 component, and not 100% sure on naming here.
but might be because I'm missing context of this change now.

Reply:
What would we want to be abstracted into 2 components? I mean the content of this component is just 2 lines of code. Do you want me to delete this component and just call the hook on where this was rendered?

Reply:
Let's go with the second option in that case.
Just to avoid confusion couple months down the line.